app/src/pages/playground/PlaygroundTool.tsx

```diff
@@ -81,6 +81,7 @@ export function PlaygroundTool({
                 instanceId: playgroundInstanceId,
                 patch: {
                   tools: instanceTools.filter((t) => t.id !== tool.id),
+                  toolChoice: undefined,
```

Comment:
why do we need this?

Reply:
exactly, the openai api throws if you send in `tool_choice` but no tools.